bigquery/datasets.js

```diff
@@ -15,123 +15,117 @@
 
 'use strict';
 
-const BigQuery = require('@google-cloud/bigquery');
+function createDataset (datasetId, projectId) {
+  // [START bigquery_create_dataset]
+  // Imports the Google Cloud client library
+  const BigQuery = require('@google-cloud/bigquery');
+
+  // The project ID to use, e.g. "your-project-id"
+  // const projectId = "your-project-id";
```

Comment:
Configuring the project ID is generally something we don't do for other products' samples. With BigQuery however, it is more common to be explicit about project ID because of public datasets. Many of the BigQuery samples already allowed configurable project ID via the CLI, I just made all the BigQuery samples consistent.